src/UI/TransactionsToastList/index.tsx

```diff
@@ -141,8 +142,12 @@ export const TransactionsToastList = ({
     ]
   );
 
-  const customToastsList = customToasts.map(
-    ({ toastId, type, duration, message = '' }: CustomToastType) => (
+  const customToastsList = customToasts
+    .filter(
+      (customToast: CustomToastType) =>
+        customToast.toastId !== TRANSACTION_STATUS_TOAST_ID
```

Comment:
What does this filter do? Can we drop a commentary?

Reply:
We are adding a new CustomToast with that specific id and we use this toast for displaying the canceled/error messages on signing a transaction. The toast is persisted in custom toasts list and we do not want to display multiple times the toast (if the TransactionToastList component is used)

Reply:
I will add a comment